### DISCUSSION TITLE
One picture per day

### DISCUSSION THREAD

**Discussion by bananajoe86** [jomjol/AI-on-the-edge-device#2896] [General]
### The Feature

Hey Guys,
I search for a posibility to safe one picture per day.
Gruß Joe

**Comment by friedpa** [comment]
Hallo Joe,
schaue Dir einmal die REST API Kommandos an. Da kannst Du mit einem Befehl ein Bild downloaden. When Du einen Server laufen hast (Linux) kann man das mit einem Script erledigen (oder z.B. über den IOBroker).

**Comment by bananajoe86** [comment]
Danke für den Hinweis. Für mich als Laie ist die Doku allerdings zu wenig beschrieben. Kannst du mir ein Beispiel geben, wie der Befehl aussehen könnte?
Danke Joe

**Comment by jomjol** [comment]
RTFM :-)  noch viel klarer kann man es kaum beschreiben:

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/30766535/ac060426-4c2b-4810-96c7-7640d47e6294)
